apps/src/blockly/addons/cdoVerticalFlyout.ts

```diff
@@ -49,22 +52,22 @@ export default class VerticalFlyout extends GoogleBlockly.VerticalFlyout {
       for (let i = 0, block; (block = blocks[i]); i++) {
         if (this.RTL) {
           // With the flyoutWidth known, right-align the blocks.
-          var oldX = block.getRelativeToSurfaceXY().x;
-          var newX = flyoutWidth / this.workspace_.scale - this.MARGIN;
+          const oldX = block.getRelativeToSurfaceXY().x;
+          let newX = flyoutWidth / this.workspace_.scale - this.MARGIN;
           if (!block.outputConnection) {
             newX -= this.tabWidth_;
           }
           block.moveBy(newX - oldX, 0);
         }
-        if (block.flyoutRect_) {
-          this.moveRectToBlock_(block.flyoutRect_, block);
+        if (this.rectMap_.has(block)) {
```

Comment:
This change comes from the [original blockly function](https://github.com/google/blockly/blob/71185b5582401cd499f0e4a8ed444e7c4d531580/core/flyout_vertical.ts#L361-L363). It has been updated since we copied it over. `block.flyoutRect_` does not exist anymore.